.changeset/perfect-parents-dress.md

```diff
@@ -0,0 +1,5 @@
+---
+'@itwin/itwinui-react': patch
+---
+
+Replaces the `width: 100%` added to `Tree` virtual items with `minInlineSize: 100%` so that virtualized `Tree` components can have horizontal scroll.
```

Comment:
Changeset could be rephrased to be less specific.

```suggestion
Fixed an issue in virtualized `Tree` where horizontal scroll was impossible to achieve.
```

Also another changeset could be added to mention the `style` prop fix.

```md
---
'@itwin/itwinui-react': patch
---
Fixed an issue in virtualized `Tree` where the `style` prop passed to `TreeNode` was not being respected.
```

Reply:
Updated changesets in this [commit](https://github.com/iTwin/iTwinUI/pull/2170/commits/55aae4785a28857c9356612b4205c2cb6445a734).